pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+Lines beginning with a hash-tag (#) may be included as comments. These and blank lines will be ignored.
+
+The file must contain the following four components in the following order: your predictions, a name for your computational protocol, a list of the major software packages used, and a long-form methods description. Each of these components must begin with a line containing only the corresponding keyword: `Predictions:`, `Name:`, `Software:`, and `Method:`, as illustrated in the example files. More detailed instructions will be provided on the challenge submission site on the D3R website.  
+
+## Evaluation strategy for computational pKa predictions
+
+Macroscopic pKa predictions will be directly compared to experimental pKa measurements. Microscopic pKas and predicted microstates will be evaluated by comparison to each other and how well they recapitulate experimental macroscopic pKas, as well as pKa predictions of ACD which uses a data-trained pKa predictions method.
+
+## Anonymous versus public participation
+
+When you upload your submission, you will have the option of having it treated anonymously. Anonymous submission means that we may report on your predictions and methods, but not your identity. Public participations means we may also say who you are. Please note that, although we will work to protect the identity of anonymous participants, we cannot make any guarantees. You may use the D3R website to change your submission’s anonymous/public status until the challenge has closed. However, after the challenge has closed, you may not change its anonymous/public status.
+
+## SAMPL6 workshop February 22-23, 2018
+
+Participants are invited to share and discuss their results, as well as the D3R and SAMPL projects more broadly, at the second in-person D3R and SAMPL workshop, which is scheduled for February 22-23, 2018, at UC San Diego, La Jolla, CA. Note that the workshop immediately follows the Biophysical Society National Meeting in San Francisco.
+
+## Files provided
+- `/physical_properties/pKa/molecule_ID_and_SMILES.csv` - CSV file that indicates SAMPL6 pKa challenge molecule IDs and canonical isomeric SMILES.
+- `/physical_properties/pKa/microstates/` - This directory contains files that list of microstate IDs and canonical isomeric SMILES of microstates. Files are separated by molecule ID.
+- `/physical_properties/pKa/submission_templates/` - Empty prediction submission template files are located here.
+- `/physical_properties/pKa/example_submission_files/` - This directory contains example submission files filled with random values to illustrate expected format.
+
+## Problems, questions and contact
+
+If you notice any issues with any of these files, please contact us via the GitHub issue tracker. You are also strongly advised to both sign up for the SAMPL e-mail list via the D3R site and sign up for notifications on this GitHub repository in case we have updates.
```

Comment:
Should this have SAMPL6 e-mail list? (I know the one for 5 was discontinued so that might make it clear you have to sign up for this challenge)

Reply:
Done.